src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -77,13 +65,23 @@ internal static bool IsPreV4Revision(BuildVersionType buildType, string id, Docu
         /// <param name="ensureStepsProcessed"></param>
         /// <param name="isLastFile"></param>
         /// <returns></returns>
-        public async Task<SmugglerResult> ExecuteAsync(bool ensureStepsProcessed = true, bool isLastFile = true)
+        public override async Task<SmugglerResult> ExecuteAsync(bool ensureStepsProcessed = true, bool isLastFile = true)
         {
             var result = _result ?? new SmugglerResult();
+            var sharded = false;
+            if (_source is DatabaseSource)
+            {
+                var record = await _source.GetDatabaseRecordAsync();
+
+                if (ShardHelper.IsShardedName(record.DatabaseName))
+                {
+                    sharded = true;
```

Comment:
this should be probably moved to source as a method

Reply:
why do we need it in this smuggler?

Reply:
we want to know if we export from shard. In that case, we won't write build version to the destination

Reply:
done